### DISCUSSION TITLE
Support Raycast

### DISCUSSION THREAD

**Discussion by alehano** [mikekazakov/nimble-commander#454] [Ideas]
Raycast has a feature to manipulate files in Finder. For instance, resize and convert images, etc. It would be great if it could do the same with files selected in Nimble Commander. I'm not sure how it works or if it is even possible. It seems it reads last active Finder window content.

**Comment by mikekazakov** [comment]
Hi, I personally do not use this software so don't know how it might work under the hood.
If there's enough interest in such a feature - the first step would be to understand how does it communicate with Finder and if something similar might be applicable for Nimble Commander.
